hasher-matcher-actioner/scripts/storm_images_queue

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+# Storms SQS images queue with messages which will be inferred as S3 uploads
+# by the pdq_hasher lambda. Given a bucket and a prefix, gets files in that
+# bucket, for each creates SQS messages and rotates through that list until it
+# has published enough messages.
+
+# If the bucket with the prefix has more than 1000 keys, only the first 1000
+# keys returned are used.
+
+# $ ./storm_images_queue <queue_url> <s3_bucket> <prefix> <count>
+# Example
+# $ ./storm_images_queue https://queue.amazonaws.com/<acid>/<queue_name> <bucket_name> images/10MB/ 100000
+
+SQS_BATCH_SIZE = 10 # matches the batch size in s3 subscription
+
+def unleash_storm(
+    queue_url: str,
+    s3_bucket: str,
+    s3_prefix: str,
+    msg_count: int
+):
+    s3_client = boto3.client('s3')
+
+    # Obtain the images
+    response = s3_client.list_objects_v2(
+        Bucket=s3_bucket,
+        Prefix=s3_prefix,
+    )
+
+    images = list(map(lambda x: x['Key'], response['Contents']))
+    image_chunks = [images[i:i + SQS_BATCH_SIZE] for i in range(0, len(images), SQS_BATCH_SIZE)]
+
+    sqs_client = boto3.client('sqs')
+
+    sent_batch_count = 0
+    sent_message_count = 0
+    while True:
+        current_chunk = image_chunks[sent_batch_count % len(image_chunks)]
+
+        # Process one chunk begin
+        sqs_client.send_message_batch(
+            QueueUrl=queue_url,
+            Entries=[{
+                        'Id': f'message-{sent_batch_count}-{i}',
+                        'MessageBody': """{"Message": "{\\\"Records\\\": [{\\\"s3\\\": {\\\"s3SchemaVersion\\\": \\\"1.0\\\",\\\"bucket\\\": {\\\"name\\\": \\\"%s\\\"},\\\"object\\\": {\\\"key\\\": \\\"%s\\\",\\\"size\\\": 12}}}]}"}""" % (s3_bucket, image)
```

Comment:
good usage of triple quotes here. 

Alt, ignorable: You can generate the same thing in a readable way by a helper method that uses json.stores() to make it easier to read.

```
entry = json.dumps({
  "MessageBody": json.dumps({
     "Records": [
       {"s3": "asdf"}
     ]
   })
})
```